packages/perseus/src/util/parse-perseus-json/perseus-parsers/numeric-input-widget.ts

```diff
@@ -35,7 +35,10 @@ export const parseNumericInputWidget: Parser<NumericInputWidget> = parseWidget(
         answers: array(
             object({
                 message: string,
-                value: optional(number),
+                // TODO(benchristel): value should never be null or undefined,
+                // but we have some content where it is anyway. If we backfill
```

Comment:
https://khanacademy.atlassian.net/wiki/spaces/ENG/pages/3524690040/Content+Issues+for+Backfill